core/modules/admin_bar/js/admin_bar.js

```diff
@@ -500,6 +500,46 @@ Backdrop.adminBar.behaviors.search = function (context, settings, $adminBar) {
   });
 };
 
+  /**
+   * Replaces the "Home" link with "Back to site" link.
+   *
+   * Back to site link points to the last non-administrative page the user visited
+   * within the same browser tab.
+   */
+  Backdrop.adminBar.behaviors.escapeAdmin = function (context, settings) {
+
+
+    if (!settings.admin_bar.back_to_site_link) {
+      return;
+    }
+
+        // Grab the stored path of the last non-admin page.
+    var escapeAdminPath = sessionStorage.getItem('escapeAdminPath');
+
+    // Saves the last non-administrative page in the browser to be able to link back
+    // to it when browsing administrative pages. If there is a destination parameter
+    // there is not need to save the current path because the page is loaded within
+    // an existing "workflow".
+    if (!settings.admin_bar.current_path_is_admin && !/destination=/.test(window.location.search)) {
+      sessionStorage.setItem('escapeAdminPath', settings.admin_bar.current_path);
+    }
+
+
+
+    // We only want to change the first anchor tag in the admin bar icon sub-menu.
+    var $toolbarEscape = $('.admin-bar-icon a').first();
+
+    // If the current page is admin, then switch the path.
+    if ($toolbarEscape.length
+      && settings.admin_bar.current_path_is_admin
+      //&& settings.admin_bar.frontpage !== escapeAdminPath
```

Comment:
So, if you prefer to always show the "Back to site" link (even when we are going back home), please remove this commented code.